README.md

```diff
@@ -86,10 +86,12 @@ Docker is a tool designed to make it easier to create, deploy, and run applicati
     ```bash
     docker build -t llm-app .
     ```
-    After your image is built, you can run it as a container. You use the docker run command to do this
+    After your image is built, you can run it as a container. You use the docker compose run command to do this
     ```bash
-    docker run -it -p 8080:8080 llm-app
+    docker compose -f deployment/docker-compose.yml run -p 8080:8080 llm-app
```

Comment:
Let's merge these 2 steps then.
```suggestion
    docker compose -f deployment/docker-compose.yml up --build
```

Reply:
This disables Text UI. It is possible to reenable it by adding: 
```
stdin_open: true
tty: true
```
to `docker-compose.yml` but then there is a "sidebar" that makes the TUI look worse than usual.

Reply:
Do you mean the `deployment-llm-app-1` prefix ?

Reply:
This is okay for me
<img width="496" alt="image" src="https://github.com/pathwaycom/llm-app/assets/123164531/52ca7c07-735c-4f68-9504-1d2229af38fe">

Reply:
Yes, and for some reason it is only on the left part of the screen. The right is empty. Because of that it is too narrow. I don't like that the connectors' names are truncated.
![image](https://github.com/pathwaycom/llm-app/assets/32928185/bd7a6d16-5d42-4f60-933d-691a522cf350)
It is not very bad, but could be better.

Reply:
Ok, if you don't like it, keep the process in 2 steps but please use docker compose for both steps.

Reply:
These work
`docker compose -f deployment/docker-compose.yml  build`
then
`docker compose -f deployment/docker-compose.yml run llm-app`